### ISSUE TITLE
Error when creating a new vm on osx

### ISSUE THREAD

**Issue by Spy-Seth** [docker/toolbox#97]
I'm trying to use toolbox but when I try to create a new machine i'm getting some error:

```
$ docker-machine create -d virtualbox --virtualbox-memory 2048 dev-test
Creating VirtualBox VM...
Creating SSH key...
Starting VirtualBox VM...
Error creating machine: exit status 1
You will want to check the provider to make sure the machine and associated resources were properly removed.
```

The created VM isn't working:
```
$ docker-machine start dev-test
exit status 1
Started machines may have new IP addresses. You may need to re-run the `docker-machine env` command.
```

I've generated a full output for debugging: https://gist.github.com/Spy-Seth/380119a55622a6e19a34
We can see this error message: `VBoxManage: error: Code NS_ERROR_INVALID_ARG (0x80070057) - Invalid argument value (extended info not available)
VBoxManage: error: Context: "RemoveRedirect(Bstr(ValueUnion.psz).raw())" at line 1766 of file VBoxManageModifyVM.cpp` 

I've the install the last toolbox version `1.8.1a`, and I've the last VirtualBox installed `5.0.2`.

**Comment by cihann** [comment]
I have same problem. I realize, "toolbox" doesn't work with virtualbox's stable version in my os x yosemite. (5.0.2/VirtualBox-5.0.2-102096) I installed virtualbox's test builds and all docker commands are just fine :)

test builds: https://www.virtualbox.org/wiki/Testbuilds

**Comment by joshdevins** [comment]
I was able to work around this by downgrading VirtualBox to the [latest stable v4.3 release](https://www.virtualbox.org/wiki/Download_Old_Builds_4_3). When looking over the Docker Machine documentation, they [mention](https://docs.docker.com/machine/get-started/) that VirtualBox 4.3 should be used.

**Comment by Spy-Seth** [comment]
So solution seem to downgrade for now. Thx @joshdevins !

**Comment by joshdevins** [comment]
I think it's still a bug though. A brand new machine (OS X 10.10), fresh download of Toolbox, and it fails to install. Seems like a pretty major bug still actually, even if there is a workaround. Can you reopen @Spy-Seth?

**Comment by Spy-Seth** [comment]
You right! I still a bug.

**Comment by oO-Zam** [comment]
Correct, the built-in VirtualBox in the installer (5.0.2) will fail. See https://forums.virtualbox.org/viewtopic.php?f=8&t=69591 for an explanation. The 5.0.3-r102137 test builds fix the issue and can be found here: https://www.virtualbox.org/wiki/Testbuilds

**Comment by jeffdm** [comment]
Thanks for the issue. The next interim version of Toolbox (1.8.1c) will include VirtualBox 5.0.0 until VBox fixes this.

Thanks!

**Comment by jeffdm** [comment]
This should be fixed in 1.8.1c which installs VirtualBox 5.0.0.

**Comment by sodabrew** [comment]
Awesome, next release will have VB 5.0.4: f3203a242df1688b1c9d868c7ab78fd2c82bbbd4 Thanks!
